src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/util/OcrDataDeserializer.java

```diff
@@ -41,7 +42,11 @@ public OcrDataDeserializer() {
         return ocrData.getFields().stream().collect(
             toMap(
                 field -> field.getName().textValue(),
-                field -> field.getValue().asText("")
+                field -> field.getValue().asText(""),
+                (u, v) -> {
+                    throw new IllegalStateException(String.format("Duplicate key for value '%s'", u));
```

Comment:
How is `IllegalStateException` handled by the main job?
Will envelope end up in an appropriate state?